README.md

```diff
@@ -1 +1,42 @@
-# test-automatiom-testNG
\ No newline at end of file
+# TestNG example
+
+Application model which describes the behavior of running the test with gradle's help
+
+## Getting Started
+
+## Running
```

Comment:
as `Getting Started` is a parent node, you may want to use 3 charp characters for nested paragraphs